bodhi/util.py

```diff
@@ -440,7 +440,13 @@ def update2html(context, update):
     else:
         title = update['title']
 
-    url = request.route_url('update', id=title)
+    alias = None
+    if hasattr(update, 'alias'):
+        alias = update.alias
+    else:
+        alias = update.get('alias')
```

Comment:
I don't fully understand this logic... if it has an alias attr, use it, else, try and get it but default to None?